x-pack/examples/ui_actions_enhanced_examples/public/containers/drilldowns_with_embeddable_example/drilldowns_with_embeddable_example.tsx

```diff
@@ -95,7 +95,7 @@ export const DrilldownsWithEmbeddableExample: React.FC = () => {
         <EuiFlexItem grow={false}>{openManagerButton}</EuiFlexItem>
         <EuiFlexItem grow={false}>
           <div style={{ maxWidth: 200 }}>
-            <ReactEmbeddableRenderer<{}, {}>
+            <EmbeddableRenderer<{}>
```

Comment:
Can you just change to `<EmbeddableRenderer`. Embeddable renderer generics already specify `SerializedState` as object so there is no need to repeat this generic.